lib/queue.js

```diff
@@ -277,35 +277,21 @@ Queue.prototype._getJobs = function (queueType, start, end, cb) {
             this.client.zrange(this.toKey(queueType), start, end, idsPromise.defer());
             break;
```

Comment:
Do something for the default cause - we don't want the promise to just never resolve.